### DISCUSSION TITLE
"Embed Sponsor button" seems to be broken?

### DISCUSSION THREAD

**Discussion by deobald** [community/community#22726] [Other Feature Feedback, Questions, & Ideas]
<p>I’ve been trying to embed the iframe provided by GitHub into my website (<a href="https://www.deobald.ca" rel="nofollow noopener">https://www.deobald.ca</a>) for a while now but I can’t seem to get it to render. Mysteriously, the iframe renders with a 0px height… though even when I turn that off in the inspector, it’s still not visible.</p>
<p>I’m not sure if this is a peculiarity of trying to render an iframe across domains, or some other security issue?</p>
<p>Has anyone else managed to get a Sponsor button embedded in an external website? I don’t see any topics on it yet.</p>
<p>Thanks!</p>
<p>PS: You <strong>will</strong> see a sponsor button on that webpage… but it’s just an image and a link. It’s not an iframe.</p>

**Comment by nholden** [comment]
<p>:wave: Hi, <a class="mention" href="/deobald">@deobald</a>!</p>
<p>Excited that you’re interested in embedding a sponsor button on your website! Sorry to hear it didn’t work on your first try.</p>
<p>Could you share the iframe code you tried to use to embed a sponsor button? Could you also share what browser you’re using?</p>
<p>In Chrome DevTools, I was able to paste this code into the sidebar of your website, and it seemed to render OK.</p>
<pre><code class="lang-auto">&lt;iframe src="https://github.com/sponsors/deobald/button" title="Sponsor deobald" height="35" width="107" style="border: 0;"&gt;&lt;/iframe&gt;
</code></pre>

**Comment by deobald** [comment]
<p>Hi <a class="mention" href="/nholden">@nholden</a>! Sorry for the slow reply… despite the lockdown in India we just moved into a new house and we’ve been a bit …internet-less. Ha ha!</p>
<p>I was using the iframe code provided by the <code>embed</code> feature verbatim:</p>
<pre><code class="lang-auto">&lt;iframe src="https://github.com/sponsors/deobald/button" title="Sponsor deobald" height="35" width="107" style="border: 0;"&gt;&lt;/iframe&gt;
</code></pre>
<p>This looks to be identical to what you’ve pasted, so it would seem the iframe code isn’t the problem?</p>
<p>Okay, very strange. I tried this on Firefox 77 (MacOS and Linux) and Chromium 83 (Linux) and Chrome 83 (MacOS). It… only seems to be broken on Firefox 77 on MacOS? Which was the only browser I was trying, initially, since that’s where my website’s build is. All the other 3 browser/OS configurations render the button perfectly.</p>
<p>I’ve put the code back onto <a href="https://www.deobald.ca" rel="nofollow noopener">https://www.deobald.ca</a> so you can have a look, if you’re interested. Firefox on MacOS renders it with a 0px height every time and I have no idea why. All the other browsers I have installed are fine.</p>
<p>Good luck, and thanks!</p>

**Comment by nholden** [comment]
<p>Thanks for sharing all those details, <a class="mention" href="/deobald">@deobald</a>!</p>
<p>I just visited <a href="https://www.deobald.ca/" rel="nofollow noopener">https://www.deobald.ca</a> on Firefox 77 on MacOS, and the button seems to be working. Here’s what I see.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/d/d056be63720e9268a1786f5ea2d4a5e87eea71ec.png" data-download-href="/uploads/short-url/tJ3a0mlg8atQLYYFHAf0Q570DiA.png?dl=1" title="Screen Shot 2020-06-28 at 8.01.37 AM" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/7942714/181072719-a119b435-ec04-4321-b00d-54a8f0056aba.png" alt="Screen Shot 2020-06-28 at 8.01.37 AM" data-base62-sha1="tJ3a0mlg8atQLYYFHAf0Q570DiA" width="690" height="491" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/d/d056be63720e9268a1786f5ea2d4a5e87eea71ec_2_690x491.png, https://aws1.discourse-cdn.com/github/optimized/2X/d/d056be63720e9268a1786f5ea2d4a5e87eea71ec_2_1035x736.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/d/d056be63720e9268a1786f5ea2d4a5e87eea71ec.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/d/d056be63720e9268a1786f5ea2d4a5e87eea71ec_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2020-06-28 at 8.01.37 AM</span><span class="informations">1270×905 212 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Do you have any browser extensions that could be blocking the iframe?</p>

**Comment by srid** [comment]
<p>The sponsor button embed is broken for me on Microsoft Edge (<code>Version 85.0.564.23 (Version officielle) beta (64 bits)</code>).</p>
<p><strong>Demo</strong>: <a href="https://neuron.zettel.page/041726b3.html" rel="nofollow noopener">https://neuron.zettel.page/041726b3.html</a></p>
<p>You will see the button on Chrome and Firefox, but not in Edge.</p>

**Comment by nholden** [comment]
<p>Hi, <a class="mention" href="/srid">@srid</a>!</p>
<p>The sponsor button embed works fine for me on Microsoft Edge (Version 84.0.522.52 (Official build) (64-bit)) for Mac, which I just downloaded this morning to test this out. Do you have any Edge Addons that could be interfering?</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/6/61ddb635a853251356dfa195f789cb009418d761.jpeg" data-download-href="/uploads/short-url/dXLks3McK0p0DTSc1j6ArXvz4NX.jpeg?dl=1" title="Screen Shot 2020-08-06 at 9.10.45 AM" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/7942714/181072726-e020ed20-7aca-4f55-85a8-a30921949763.jpeg" alt="Screen Shot 2020-08-06 at 9.10.45 AM" data-base62-sha1="dXLks3McK0p0DTSc1j6ArXvz4NX" width="657" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/6/61ddb635a853251356dfa195f789cb009418d761_2_657x500.jpeg, https://aws1.discourse-cdn.com/github/optimized/2X/6/61ddb635a853251356dfa195f789cb009418d761_2_985x750.jpeg 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/6/61ddb635a853251356dfa195f789cb009418d761_2_1314x1000.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/6/61ddb635a853251356dfa195f789cb009418d761_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2020-08-06 at 9.10.45 AM</span><span class="informations">2164×1646 314 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by deobald** [comment, marked as answer]
<p>Hey <a class="mention" href="/nholden">@nholden</a>! Thanks for that. Sorry for the slow reply. It does look like it’s AdBlocker Ultimate that’s causing the issue, even though I have a whitelist entry for <code>github.com</code>. It feels a little aggressive for ad blockers to consider sponsorship buttons “advertisements” but I suppose I’m really not one to judge here. :wink:</p>
<p>Thanks for the help!</p>
